### PR TITLE
Add support for Minecraft 1.17

### PR DESCRIPTION
**Pull request by AndreVuillemot160**
The pull request resolves the issue #35
I did tests yesterday, and everything works as intended,

### CODE REVIEW

File: MineColab.ipynb [hunk 1]
```diff
@@ -58,16 +58,16 @@
         "from pyngrok import conf, ngrok\n",
         "# Update the package lists\n",
         "!sudo apt update &>/dev/null && echo \"apt cache successfully updated\" || echo \"apt cache update failed, you might receive stale packages\"\n",
-        "# Install OpenJDK 15\n",
+        "# Install OpenJDK 16\n",
         "!wget -qO - https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public | sudo apt-key add -\n",
-        "!sudo add-apt-repository --yes https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/ &>/dev/null || echo \"Failed to add repo. Still can be ignored if openjdk15 gets installed.\"\n",
-        "!sudo apt-get install adoptopenjdk-15-hotspot-jre &>/dev/null && echo \"Yay! Openjdk15 has been successfully installed.\" || echo \"Failed to install OpenJdk15.\"\n",
+        "!sudo add-apt-repository --yes https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/ &>/dev/null || echo \"Failed to add repo. Still can be ignored if openjdk16 gets installed.\"\n",
+        "!sudo apt-get install adoptopenjdk-16-hotspot-openj9 &>/dev/null && echo \"Yay! Openjdk16 has been successfully installed.\" || echo \"Failed to install OpenJdk16.\"\n",
```
Comment (thecoder-001) [inline MineColab.ipynb:64]: Umm...there's no jvm named `adoptopenjdk-16-hotspot-openj9`. Both hotspot and openj9 exist as independent jvms.

File: MineColab.ipynb [hunk 2]
```diff
@@ -125,8 +125,9 @@
         "id": "Fy5-YjJMPV3S"
       },
       "source": [
-        "# Replace \"1.16.4\" with your desired server version.\n",
+        "# Replace \"1.17\" with your desired server version.\n",
         "# Available versions:\n",
+        "# - 1.17\n",
```
Comment (thecoder-001) [inline MineColab.ipynb:130]: Maybe adding "experimental builds, use with caution" would be desirable as paper's 1.17 servers are still in dev.
image

**Review by thecoder-001 — CHANGES_REQUESTED** [review]

File: MineColab.ipynb (hunk 1 above)
Comment (Shadow0Bit) [reply MineColab.ipynb:64]: It should be `adoptopenjdk-16-openj9`, propably

Comment (AndreVuillemot160) [reply MineColab.ipynb:64]: > Umm...there's no jvm named `adoptopenjdk-16-hotspot-openj9`. Both hotspot and openj9 exist as independent jvms.
> Maybe a typo while making the pr.
 
oops

**Comment by AndreVuillemot160** [comment]
Done

**Comment by thecoder-001** [comment]
Sorry for taking so long. My previous test instance was somehow causing a protection fault in the jvm and blowing it to pieces. Looks to be working fine now after fiddling around with some stuff.
Thanks for your pr!

**Review by thecoder-001 — APPROVED** [review]
Sorry for taking so long. My previous test instance was somehow causing a protection fault in the jvm and blowing it to pieces. Looks to be working fine now after fiddling around with some stuff.
Thanks for your pr!

**Pull request merged**